tests/unit/amo/pages/ErrorPages/TestUnavailableForLegalReasonsPage.js

```diff
@@ -1,13 +1,37 @@
-import { shallow } from 'enzyme';
-import * as React from 'react';
-
-import UnavailableForLegalReasons from 'amo/components/Errors/UnavailableForLegalReasons';
-import UnavailableForLegalReasonsPage from 'amo/pages/ErrorPages/UnavailableForLegalReasonsPage';
+import {
+  createHistory,
+  renderPage as defaultRender,
+  screen,
+} from 'tests/unit/helpers';
 
 describe(__filename, () => {
-  it('renders a NotFound component', () => {
-    const root = shallow(<UnavailableForLegalReasonsPage />);
+  it('renders an UnavailableForLegalReasons Page', () => {
```

Comment:
I've read this diff multiple times but I can't see where it actually triggers the `UnavailableForLegalReasonsPage` component/page be shown?

Reply:
We are navigating to `/en-US/firefox/451/` (by setting `initialEntries` when creating `history` during `render`), and that is a special location that will show the `UnavailableForLegalReasonsPage` (see https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/Routes/index.js#L199-L203).